core/cmd/client_test.go

```diff
@@ -136,8 +134,7 @@ func TestTerminalAPIInitializer_InitializeWithoutAPIUser(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			store, cleanup := cltest.NewStore(t)
-			defer cleanup()
+			store := cltest.NewStore(t)
```

Comment:
Bonus points if you replace it with `pgtest.NewGormDB`

Reply:
I think @archseer is doing that in parallel - some already rebased away.

Reply:
Looks like waiting for #5013 to merge would trim a lot more noise from this PR.